fv3core/stencils/d_sw.py

```diff
@@ -34,7 +33,7 @@ def grid():
 
 
 def k_bounds():
-    return [[0, 1], [1, 1], [2, 1], [3, grid().npz - 3]]
+    return [[0, 1], [1, 1], [2, 1], [3, grid().npz - 2]]
```

Comment:
I don't understand why this changed?

Reply:
UpdatedzD needs to go one k level higher than D_SW, to the buffer point that usually isn't used. To reuse the same 'column_namelist' and remove the specification of 'kstart' and 'nk in many methods, I just make all of the column namelist calculations go to the top of the array

Reply:
understood, thanks